zqd/space/space.go

```diff
@@ -111,7 +121,10 @@ func loadSpaces(p iosrc.URI, conf config) ([]Space, error) {
 	if datapath.IsZero() {
 		datapath = p
 	}
-
+	pcapstore, err := loadPcapStore(datapath)
```

Comment:
Should this move into the `case storage.FileStore` block, since the archive space won't actually do anything with the pcap yet?

Reply:
spaceBase depends on pcapstore being not nil, but I could of course add a check for this.